model-optimizer/mo/utils/graph.py

```diff
@@ -33,7 +36,8 @@ def backward_bfs_for_operation(start_node: Node, op_names: list):
                     if pnode.id not in ret:
                         ret.append(pnode.id)
                 else:
-                    q.append(pnode)
+                    if pnode.op not in skip_op_list:
+                        q.append(pnode)
```

Comment:
shouldn't we break `for` loop if an operation from this list is met based on description above?